hles/logs/errors.log

```diff
@@ -0,0 +1,9 @@
+{"errorType":"MissingOwnerIdError","message":"Record 1 has less than 1 value for field st_owner_id"}
```

Comment:
⛔ let's not check this file in

Reply:
yeah whoops, it should have pushed it out